cmd/api/src/model/jobs.go

```diff
@@ -108,15 +108,15 @@ func (s FileUploadJobs) GetValidFilterPredicatesAsStrings(column string) ([]stri
 type JobStatus int
 
 const (
-	JobStatusInvalid   JobStatus = -1
-	JobStatusReady     JobStatus = 0
-	JobStatusRunning   JobStatus = 1
-	JobStatusComplete  JobStatus = 2
-	JobStatusCanceled  JobStatus = 3
-	JobStatusTimedOut  JobStatus = 4
-	JobStatusFailed    JobStatus = 5
-	JobStatusIngesting JobStatus = 6
-	JobStatusAnalyzing JobStatus = 7
+	JobStatusInvalid           JobStatus = -1
+	JobStatusReady             JobStatus = 0
+	JobStatusRunning           JobStatus = 1
+	JobStatusComplete          JobStatus = 2
+	JobStatusCanceled          JobStatus = 3
+	JobStatusTimedOut          JobStatus = 4
+	JobStatusFailed            JobStatus = 5
+	JobStatusIngesting         JobStatus = 6
+	JobStatusAnalyzing         JobStatus = 7
 	JobStatusPartiallyComplete JobStatus = 8
```

Comment:
hmm im going to check out why my editor didnt format this! I think I may have been the last person to touch this file